packages/vuetify/src/components/VTimePicker/VTimePickerClock.sass

```diff
@@ -32,6 +32,7 @@
   &__container
     display: flex
     flex-direction: column
+    flex-basis: 290px
```

Comment:
Alternatively, we could remove all flexbox-related properties since this will probably won't have any effect. Deleted code is the best code. 🏀 

```suggestion
```

Reply:
290px is just the default value, does it work with different widths? and with full-width?
I'll check it later anyway (but only on current browsers)

Reply:
Thanks @jacekkarczmarczyk! It seems to work fine on all browsers and widths unless I'm missing something component-specific. Let me know if any further changes are needed. 🔍

Reply:
This fix is concise and resolves the issue. More global changes should be iterated on in a separate PR.

Thank you!

Reply:
Thanks @johnleider @jacekkarczmarczyk! 🙏